### ISSUE TITLE
using ananconda to manage packages?

### ISSUE THREAD

**Issue by biocyberman** [chasemc/electricShine#394]
Hi
This looks like an interesting project, and I want to have support for Linux.

> It currently only builds windows apps, but I'm investiging adding support for Mac and Linux as well. That is quite a bit harder because the R installation for Mac and Linux hard-codes paths into the installation and part of the benefit of {electricShine} is not relying on the system's version of R

I've been using anaconda on MacOS, Linux, and Windows. It's very stable and portable to manage python, R, nodejs, and dependencies. So would it be easier to wrap electricShine around it?

**Comment by chasemc** [comment]
Do you mean to use Conda within electricShine as a dependency manager?

**Comment by biocyberman** [comment]
Exactly. Nodejs and its dependencies can be managed with conda, so you don't need to worry about where and how to install things.

**Comment by chasemc** [comment]
I like the idea mainly because I'm still considering how to make a version/option where R packages are first downloaded on the end-users' machine to avoid licensing issues.  The problem I foresee is when a package requires Rtools on a Windows machine?

**Comment by chasemc** [comment]
see https://github.com/chasemc/electricShine/projects
